packages/eslint-config-commons-ui/next.js

```diff
@@ -1,3 +1,11 @@
-module.exports = {
-  extends: [require.resolve("."), "plugin:@next/next/core-web-vitals"],
-};
+const { fixupConfigRules } = require("@eslint/compat");
+const { FlatCompat } = require("@eslint/eslintrc");
+
+const commonConfig = require("./index");
+
+const flatCompat = new FlatCompat();
+
+module.exports = [
+  ...fixupConfigRules(flatCompat.extends("plugin:@next/next/core-web-vitals")),
```

Comment:
Why do we need this? Aren't we [going](https://nextjs.org/blog/next-15#eslint-9-support) all in [Eslint 9](https://eslint.org/blog/2024/04/eslint-v9.0.0-released/#flat-config-is-now-the-default-and-has-some-changes)?

Reply:
@kilemensi I came across [this issue](https://github.com/vercel/next.js/issues/64114) and the only workaround I had was to use [eslint compatibility utilities](https://eslint.org/blog/2024/05/eslint-compatibility-utilities/) to fix up things.